src/Projection/PdoEventStoreReadModelProjector.php

```diff
@@ -151,6 +161,7 @@ public function __construct(
         $this->persistBlockSize = $persistBlockSize;
         $this->sleep = $sleep;
         $this->status = ProjectionStatus::IDLE();
+        $this->triggerPcntlSignalDispatch = $triggerPcntlSignalDispatch && extension_loaded('pcntl');
```

Comment:
if $triggerPcntlSignalDispatch is set to true but extension is missing, we should maybe throw an exception instead.